code/modules/vending/clothingbooth_items.dm

```diff
@@ -46,6 +46,90 @@
 	category = "Accessories"
 	cost = PAY_TRADESMAN/3
 
+/datum/clothingbooth_item/hbowmagenta
+	name = "Magenta Hair Bow"
+	path = /obj/item/clothing/head/hairbow/magenta
+	slot = SLOT_HEAD
+	category = "Accessories"
+	cost = PAY_TRADESMAN/3
```

Comment:
Thought:
I imagine this was annoying to type out. A medium-sized #define could be used in the future to save on some typing.
First example that came to mind: (it's a doozy of an example)
https://github.com/goonstation/goonstation/blob/f173e5ebf7fe6952444e8f81b88811f686b696dc/code/modules/power/cablecolors.dm#L1

Reply:
Yeah, that's definitely something I'll keep in mind for the future. This PR I got to learn about the magical world of child objects, so hopefully next time I can try that!